src/containers/nft-view/NFTView.tsx

```diff
@@ -0,0 +1,40 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Features Imports
+import NFTViewStats from '../../features/ui/Stats/NFTViewStats';
+
+//- Hooks Imports
+import { useBidData } from '../../lib/hooks/useBidData';
+import { useDomainEvents } from '../../lib/hooks/useDomainEvents';
+
+//- Library Imports
+import { Domain, DomainMetrics, TokenPriceInfo } from '@zero-tech/zns-sdk';
+
+type NFTViewContainerProps = {
+	domain: Domain;
+	metrics: DomainMetrics;
+	paymentTokenInfo: TokenPriceInfo;
+};
+
+const NFTViewContainer: FC<NFTViewContainerProps> = ({
```

Comment:
added some NFT view page content to bring through the correct Bid data for stats widget

Reply:
Nice work!